src/bot/commands/command-play.ts

```diff
@@ -45,23 +25,47 @@ export class Play extends Command {
         guildId: String(voiceMember.guild.id),
       });
 
-      const player = this.getPlayer(voiceMember.channelId);
-      connection.subscribe(player);
+      const source = this.getSourceStream();
 
       const queue = this.getQueue();
-      queue.add(voiceMember.channelId, {
-        audioResource,
-        streamInfo,
-        userSearch: input,
+
+      const video = await source.getStreamFromUrl(input);
+
+      const searchResult =
```

Comment:
Why not name this StreamInfoCollection, since its type is StreamInfo[]

Reply:
resolved: [5252756](https://github.com/Marli-Marlete/marli-music/pull/21/commits/5252756b4ba49d970306720a3bb9a66014cda2c6)